contributors/devel/help-wanted.md

```diff
@@ -1,13 +1,118 @@
+We use two labels `help wanted` and `good first issue` to identify issues that have been
```

Comment:
This ends up looking weird at the top as there is now no title header on the page (https://github.com/carolynvs/community/blob/65bc92febd51f0ca93868054a410ede35d9f0c17/contributors/devel/help-wanted.md). Maybe an "Overview" type heading?